src/wallet/wallet_rpc_server.cpp

```diff
@@ -2439,6 +2439,126 @@ namespace tools
     return true;
   }
   //------------------------------------------------------------------------------------------------------------------------------
+  bool wallet_rpc_server::on_get_transfers_csv(const wallet_rpc::COMMAND_RPC_GET_TRANSFERS_CSV::request& req, wallet_rpc::COMMAND_RPC_GET_TRANSFERS_CSV::response& res, epee::json_rpc::error& er, const connection_context *ctx)
+  {
+    wallet_rpc::COMMAND_RPC_GET_TRANSFERS::response response;
+    if (!on_get_transfers(req, response, er, ctx))
+    {
+      return false;
+    }
+
+    std::stringstream output;
```

Comment:
wallet_rpc_server has access to wallet2, so it feels like export_transfers and this should be put in there, otherwise we're definitely going to forget to update one and not the other

Reply:
Yeah I thought the wallet and the rpc codes needed consolidation.
Currently there is `wallet_rpc_server::on_get_transfers` and `simple_wallet::get_transfers` that are doing basically the same thing.
Definitely we should have the rpc use the wallet functions, I just thought that it could be refactored later.